### ISSUE TITLE
[scss] if selector ends with operator two spaces are put between it and bracket

### ISSUE THREAD

**Issue by zieka** [prettier/prettier#3731]
**Prettier 1.9.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA6GALAlgZwAQB8eqCAJvkagO5aaHECGU9qEADnAE4MwSf3AAOszwB6UXigR4eTHDykIAVwBGAG3k42DMPJVwYVOAlkZ57Ljz54mpPCu5gA1geEBfYcPTYKxMr+paDBYmFgtuXn4AajwhEXFJaXk5BWV1TW1dewMjExTwq35be0cXGHdPKG9cFn8WGjpKUMoCyLwAP1jhPDEJKRkUxVUNPC0dPRzjZnyOCOtihx0y9xAAGhB2GCxoHGRQBk5OCCoABQOEXZQGADcILFI1kAYcGGQAMwY1HDh1xecDADKmSwUAA5sgYJwlD8QIowO9Pt91iDvpwYCduKCALYMBFfGEAKxwAA8AEKlQEMLFwAAyILgeKRGyUMDYLIATIyYdpOKjkCAcGAcLt1mxOCCYAB1e6YZAADgADKKjt9Jdw2PyxXBUdcGetOHAAI5KLAGjEMbG4pAffHrb5YrAQqEwnAg0EaACKSiSXPWMAYKmlpFlSAALH7uFg1G6AMIQLE4-lSKB6kBKb4AFQDl0h0LcbiAA)
```sh
--parser scss
--tab-width 4
--use-tabs
```

**Input:**
```scss
.this > .ends > .with > .an > .operator > {
  // note the double space between the operator and bracket
}

.this > .ends > .with > .an > .operator + {
  // note the double space between the operator and bracket
}

.this > .ends > .with > .an > .operator ~ {
  // note the double space between the operator and bracket
}
```

**Output:**
```scss
.this > .ends > .with > .an > .operator >  {
	// note the double space between the operator and bracket
}

.this > .ends > .with > .an > .operator +  {
	// note the double space between the operator and bracket
}

.this > .ends > .with > .an > .operator ~  {
	// note the double space between the operator and bracket
}

```

**Expected behavior:**
Expected there to only be one space between the last operator/combinator in the selector and the bracket (currently there are two spaces)

**Comment by vjeux** [comment]
I'm curious, what does it even mean?

**Comment by zieka** [comment]
while not the greatest use of the syntax scss will let you end a selector with an operator/combinator
So this **scss**:
```scss
.parent > {
  .child-one {
    color: red;
  }
  .child-two {
    color: blue;
  }
}
```
compiles to following **css**
```css
.parent > .child-one {
  color: red;
}
.parent > .child-two {
  color: blue;
}
```

in my opinion, the proper formatting for this should be the following **scss**:
```scss
.parent {
  > .child-one {
    color: red;
  }
  > .child-two {
    color: blue;
  }
}
```
but for now the double space printing is ugly.

**Comment by lydell** [comment]
Wow, I had no idea that was a feature! TIL

**Comment by vjeux** [comment]
@zieka thanks! I had no idea it existed. We should definitely not add two spaces. We already have a special-case for the second version which is more common.

**Comment by ikatyang** [comment]
For reference, fixed in #3738.
